cmd/internal/ecs/ecs.go

```diff
@@ -93,17 +112,21 @@ func init() {
 	Cmd.PersistentFlags().MarkHidden("aws-credentials")
 
 	Cmd.AddCommand(AddToECSCmd)
+	Cmd.AddCommand(PrintCloudFormationFragmentCmd)
+	Cmd.AddCommand(PrintECSTaskDefinitionCmd)
 	Cmd.AddCommand(RemoveFromECSCmd)
 }
 
-func addAgentToECS(cmd *cobra.Command, args []string) error {
-	// Check for API key
+// Checks that an API key and a project ID are provided, and that the API key is
+// valid for the project ID.
+func checkAPIKeyAndProjectID() error {
+	// Check for API key.
 	_, err := cmderr.RequirePostmanAPICredentials("The Postman Insights Agent must have an API key in order to capture traces.")
 	if err != nil {
 		return err
 	}
 
-	// Check collecton Id's existence
+	// Check that a collection or project is provided.
 	if collectionId == "" && projectId == "" {
 		return errors.New("exactly one of --project or --collection must be specified")
 	}
```

Comment:
I'm not clear why we need to continue to support --collection in the ecs subcommands.  Seems like we could deprecate them as long as it is still supported in 'apidump' for those who haven't modified their tasks yet.

Reply:
Removed in f346cb7.